.gitignore

```diff
@@ -1,7 +1,10 @@
 /.idea
 /node_modules
 /.vscode
-/dist
```

Comment:
dist should be fully ignored in my opinion

Reply:
Updated.